mycore-ocfl/src/main/java/org/mycore/ocfl/commands/MCROCFLCommands.java

```diff
@@ -57,6 +56,14 @@ public class MCROCFLCommands {
 
     public static final String FAILED_AND_NOW_INVALID_STATE = FAILED + " and now invalid state";
 
+    private static final MCROCFLXMLUserManager OCFL_USER_MANAGER
+        = MCRConfiguration2.<MCROCFLXMLUserManager>getSingleInstanceOf("MCR.OCFL.User.Manager")
+            .orElseThrow();
+
+    private static final MCROCFLXMLClassificationManager OCFL_CLASSIFICATION_MANAGER
+        = MCRConfiguration2.<MCROCFLXMLClassificationManager>getSingleInstanceOf("MCR.OCFL.Classification.Manager")
+            .orElseThrow();
+
```

Comment:
better provide private getter methods for this. This could cause the CLI to break globally